src/include/sof/audio/ipc-config.h

```diff
@@ -25,6 +26,10 @@ struct ipc_config_dai {
 	uint32_t direction;	/**< SOF_IPC_STREAM_ */
 	uint32_t dai_index;	/**< index of this type dai */
 	uint32_t type;		/**< DAI type - SOF_DAI_ */
+	/* physical protocol and clocking */
+	uint16_t format;	/**< SOF_DAI_FMT_ */
+	uint16_t group_id;	/**< group ID, 0 means no group (ABI 3.17) */
```

Comment:
@lgirdwood why do we need the group_Id here? it is already part of spec_config anyway no?

Reply:
It's common infrastructure applicable to any DAI type, i.e. we can group several SSPs or DMICs